decidim-admin/app/queries/decidim/admin/manageable_participatory_processes_for_user.rb

```diff
@@ -3,7 +3,13 @@ module Decidim
   module Admin
     # A class used to find the ParticipatoryProcesses that the given user can
     # manage.
-    class ManageableParticipatoryProcessesForUser < Rectify::Query
+    class ManageableParticipatoryProcessesForUser
```

Comment:
In order to enforce symmetry and enforce common good practices, I think this should better be a Rectify::Query. It clearly follows its semantics - gets a set of inputs and has a single output method.